diff --git a/lib/cjs/api/sdk.js b/lib/cjs/api/sdk.js
index fba0381d5f15a3f586986db7fd8c95b3955226a7..e30c1a55ed8b25660a545d1b3302b169a4dcece2 100644
--- a/lib/cjs/api/sdk.js
+++ b/lib/cjs/api/sdk.js
@@ -7,14 +7,14 @@ exports.GetMarketsDocument = (0, graphql_tag_1.gql) `
     query getMarkets($chainId: Int!, $marketIds: [String!]) {
   markets(where: {chainId_in: [$chainId], uniqueKey_in: $marketIds}) {
     items {
-      uniqueKey
+      uniqueKey: marketId
       targetBorrowUtilization
       publicAllocatorSharedLiquidity {
         vault {
           address
         }
-        allocationMarket {
-          uniqueKey
+        allocationMarket: withdrawMarket {
+          uniqueKey: marketId
         }
         assets
       }
@@ -23,7 +23,7 @@ exports.GetMarketsDocument = (0, graphql_tag_1.gql) `
         state {
           allocation {
             market {
-              uniqueKey
+              uniqueKey: marketId
               loanAsset {
                 address
               }
diff --git a/lib/esm/api/sdk.js b/lib/esm/api/sdk.js
index 42393261b069a582e54f4028b235451c008f9f5b..b30fbee68dab2b6289f2d96b170a68a79200fa00 100644
--- a/lib/esm/api/sdk.js
+++ b/lib/esm/api/sdk.js
@@ -3,14 +3,14 @@ export const GetMarketsDocument = gql `
     query getMarkets($chainId: Int!, $marketIds: [String!]) {
   markets(where: {chainId_in: [$chainId], uniqueKey_in: $marketIds}) {
     items {
-      uniqueKey
+      uniqueKey: marketId
       targetBorrowUtilization
       publicAllocatorSharedLiquidity {
         vault {
           address
         }
-        allocationMarket {
-          uniqueKey
+        allocationMarket: withdrawMarket {
+          uniqueKey: marketId
         }
         assets
       }
@@ -19,7 +19,7 @@ export const GetMarketsDocument = gql `
         state {
           allocation {
             market {
-              uniqueKey
+              uniqueKey: marketId
               loanAsset {
                 address
               }
